### ISSUE TITLE
Segfault, oh my!

### ISSUE THREAD

**Issue by swails** [Amber-MD/cpptraj#419]
```
$ echo "help analyze" | bin/cpptraj

CPPTRAJ: Trajectory Analysis. V17.00b
    ___  ___  ___  ___
     | \/ | \/ | \/ | 
    _|_/\_|_/\_|_/\_|_

| Date/time: 11/17/16 12:44:16
| Available memory: 8.488 GB

INPUT: Reading input from 'STDIN'
  [help analyze]
Segmentation fault
```

:smiley:

**Comment by drroe** [comment]
Should be fixed by #420. Good catch, thanks @swails!

**Comment by swails** [comment]
Sweet!
